lib/internal/Magento/Framework/Module/PackageInfo.php

```diff
@@ -85,9 +95,9 @@ private function load()
                 $key = $moduleDir . '/composer.json';
                 if (isset($jsonData[$key]) && $jsonData[$key]) {
                     try {
-                        $packageData = \Zend_Json::decode($jsonData[$key]);
-                    } catch (\Zend_Json_Exception $e) {
-                        throw new \Zend_Json_Exception(
+                        $packageData = $this->serializer->unserialize($jsonData[$key]);
+                    } catch (\InvalidArgumentException $e) {
```

Comment:
I am not a big fan of catching this and then throwing the same exception. I was thinking about adding a new exception for this case here. What are people's thoughts on this?